products/rhel10/profiles/stig_gui.profile

```diff
@@ -7,14 +7,14 @@ metadata:
 
 reference: https://public.cyber.mil/stigs/downloads/?_dl_facet_stigs=operating-systems%2Cunix-linux
 
-title: 'DRAFT - DISA STIG for Red Hat Enterprise Linux 10'
+title: 'Red Hat STIG for Red Hat Enterprise Linux 10'
 
 description: |-
-    This is a draft profile for experimental purposes.
+    This is a profile based on what is expected in the RHEL 10 STIG.:
     It is not based on the DISA STIG for RHEL 10, because it was not available at time of
     the release.
 
-    In addition to being applicable to Red Hat Enterprise Linux 10, DISA recognizes this
+    In addition to being applicable to Red Hat Enterprise Linux 10, this
```

Comment:
Now the sentence doesn't make sense.

I think you can change "as" to "is" and then it will be fine.